pymongo/encryption.py

```diff
@@ -682,6 +704,74 @@ def encrypt(
                 key_alt_name=key_alt_name,
                 query_type=query_type,
                 contention_factor=contention_factor,
+                range_opts=range_options,
+            )
+            return decode(encrypted_doc)["v"]  # type: ignore[index]
+
+    def encrypt_expression(
+        self,
+        expression: Mapping[str, Any],
+        algorithm: str,
+        key_id: Optional[Binary] = None,
+        key_alt_name: Optional[str] = None,
+        query_type: Optional[str] = None,
+        contention_factor: Optional[int] = None,
+        range_options: Optional[EncryptionRangeOpts] = None,
+    ) -> RawBSONDocument:
+        """Encrypt a BSON expression with a given key and algorithm.
+
+        Note that exactly one of ``key_id`` or  ``key_alt_name`` must be
+        provided.
+
+        :Parameters:
+          - `expression`: The BSON aggregate or match expression to encrypt.
+          - `algorithm` (string): The encryption algorithm to use. See
+            :class:`Algorithm` for some valid options.
+          - `key_id`: Identifies a data key by ``_id`` which must be a
+            :class:`~bson.binary.Binary` with subtype 4 (
+            :attr:`~bson.binary.UUID_SUBTYPE`).
+          - `key_alt_name`: Identifies a key vault document by 'keyAltName'.
+          - `query_type` (str): **(BETA)** The query type to execute. See
+            :class:`QueryType` for valid options.
+          - `contention_factor` (int): **(BETA)** The contention factor to use
+            when the algorithm is :attr:`Algorithm.INDEXED`.  An integer value
+            *must* be given when the :attr:`Algorithm.INDEXED` algorithm is
+            used.
+          - `range_options`: An instance of EncryptionRangeOptions.
```

Comment:
We should use the `**BETA**` consistently here and the other places we mention the new feature.